physionet-django/environment/services.py

```diff
@@ -36,15 +36,15 @@ def _environment_dataset(environment: ResearchEnvironment) -> str:
 
 def create_cloud_identity(user: User) -> Tuple[str, CloudIdentity]:
     response = api.create_cloud_identity(
-        f"researcher.{user.username}", user.profile.first_names, user.profile.last_name
+        f"researcher_{user.username}", user.profile.first_names, user.profile.last_name
```

Comment:
I'd move this to a variable and use it here and in line 47 (I know it was me who originally wrote it the wrong way :P):

```python
gcp_user_id = f"researcher_{user.username}"
response = api.create_cloud_identity(gcp_user_id, user.profile.first_names, user.profile.last_name)
...
...
identity = CloudIdentity.objects.create(user=user, gcp_user_id=gcp_user_id, email=body["email-id"])
```